File: AudioPlay/pre/EOCAudioClassThree/EOCAudioClassOne/ViewController.m
//
//  ViewController.m
//  EOCAudioClassOne
//
//  Created by EOC on 2017/6/28.
//  Copyright © 2017年 EOC. All rights reserved.
//

#import "ViewController.h"
#import "EOCAudioPlayerViewCtr.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    
    [super viewDidLoad];

 
}



- (void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event{
    
    EOCAudioPlayerViewCtr *eocAudioPlayerViewCtr = [EOCAudioPlayerViewCtr new];
    
    [self presentViewController:eocAudioPlayerViewCtr animated:YES completion:nil];
    
}






@end

File: NavAnimation/NavAnimation/ViewController.swift
//
//  ViewController.swift
//  NavAnimation
//
//  Created by Casey on 11/01/2019.
//  Copyright © 2019 newplatform. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }


    
    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
        
        let nav = AnimationNavigationCtr.init(rootViewController: NavAdvanceAnimaViewCtr())
        
        self.present(nav, animated: true, completion: nil)
    
    }
}

